sdk/identity/Azure.Identity/samples/ClientSideUserAuthentication.md

```diff
@@ -84,22 +96,29 @@ Here is an example of an application storing the `AuthenticationRecord` to the l
 ```C# Snippet:Identity_ClientSideUserAuthentication_Persist_AuthRecord
 AuthenticationRecord authRecord = await credential.AuthenticateAsync();
 
-using var authRecordStream = new FileStream(AUTH_RECORD_PATH, FileMode.Create, FileAccess.Write);
-
-await authRecord.SerializeAsync(authRecordStream);
-
-await authRecordStream.FlushAsync();
+using (var authRecordStream = new FileStream(AUTH_RECORD_PATH, FileMode.Create, FileAccess.Write))
+{
+    await authRecord.SerializeAsync(authRecordStream);
```

Comment:
Is flush not needed here? The docs for [FileStream.Dispose](https://docs.microsoft.com/en-us/dotnet/api/system.io.filestream.dispose?view=net-5.0) don't mention flushing the buffer.

Reply:
Jared confirmed that it is not needed.  Would you like me to research it more?